styleguide/components/21-atoms/heading/heading.config.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
 module.exports = {
-  collated: true,
   collator: function (markup, item) {
     return `<!-- Start: @${item.handle} -->\n<dt>${item.name}</dt><dd style="max-width: 100%">${markup}</dd>\n<!-- End: @${item.handle} -->\n`;
```

Comment:
The max-width is legacy, it's covered in styleguide css.
You can remove these collator functions too.